boot/helm-charts/slimeboot/templates/deployment.yaml

```diff
@@ -97,11 +97,17 @@ spec:
           volumeMounts:
             - mountPath: /etc/slime/config
               name: config-volume
+            {{- with $.Values.containers.volumeMounts }}
```

Comment:
containers? it looks like an array

Reply:
chang to $.Values.containers.slime.volumeMounts